KPP/carbon/carbon_Funcs.F90

```diff
@@ -391,87 +390,54 @@ END FUNCTION carbon_Get_COfromNMVOC_Flux
 !\\
 !\\
 ! !INTERFACE:
 !
-  FUNCTION carbon_Get_CO2fromOH_Flux( dtChem ) RESULT ( flux )
-!
-! !INPUT PARAMETERS:
-!
-    REAL(dp), INTENT(IN) :: dtChem       ! Chemistry timestep [s]
-!
-! !RETURN VALUE:
-!
-    REAL(dp)             :: flux         ! CO2_OH flux [molec/cm3/s]
-
-!EOP
-!------------------------------------------------------------------------------
-!BOC
-
-    flux = C(ind_CO2fromOH) / dtChem     ! molec/cm3 --> molec/cm3/s
+  FUNCTION GC_OHCO() RESULT( k )
 
-  END FUNCTION carbon_Get_CO2fromOH_Flux
-!EOC
-!------------------------------------------------------------------------------
-!                  GEOS-Chem Global Chemical Transport Model                  !
-!------------------------------------------------------------------------------
-!BOP
 !
-! !IROUTINE: carbon_get_oh_e_flux
+! !REMARKS:
+! Tropospheric loss of CO due to chemical rxn w/ OH
 !
-! !DESCRIPTION: Returns the flux of OH_E in molec/cm3/s for diagnostics.
-!\\
-!\\
-! !INTERFACE:
+!  DECAY RATE
+!  The decay rate (KRATE) is calculated by:
 !
-  FUNCTION carbon_Get_FixedOH_Flux( dtChem ) RESULT ( flux )
+!     OH + CO -> products (JPL 15-10)
+!     k = (1 + 0.6Patm) * 1.5E-13
 !
-! !INPUT PARAMETERS:
+!  KRATE has units of [ molec^2 CO / cm6 / s ]^-1,
+!  since this is a 2-body reaction.
 !
-    REAL(dp), INTENT(IN) :: dtChem       ! Chemistry timestep [s]
+! From JPL 2006: "The  reaction between HO and CO to yield
+! H + CO2 akes place on a potential energy surface that
+! contains the radical HOCO.  The yield of H and CO2 is
+! diminished as the pressure rises.  The loss of reactants
+! is thus the sum of two processes, an association to yield
+! HOCO and the chemical activation process yielding H and
+! CO2." So we now need two complicated reactions.
 !
-! !RETURN VALUE:
+! GY( A0 = 5.9e-33, B0 = 1.,     A1 = 1.1e-12, B1 = -1.3e0,
+!     A2 = 1.5e-13, B2 = 0.,     A3 = 2.1e09,  B3 = -6.1e0 )
 !
-    REAL(dp)             :: flux         ! CO2_OH flux [kg/m2/s]
-
 !EOP
 !------------------------------------------------------------------------------
 !BOC
-
-    flux = C(ind_FixedOH) / dtChem       ! molec/cm3 --> molec/cm3/s
-
-  END FUNCTION carbon_Get_FixedOH_Flux
-
-
-  !==========================================================================
-  ! Rate-law functions
-  !==========================================================================
-
-  FUNCTION GC_OHCO() RESULT( k )
-    !
-    ! Reaction rate for:
-    !    OH + CO = HO2 + CO2 (cf. JPL 15-10)
-    !
-    ! For this reaction, these Arrhenius law terms evaluate to 1:
-    !    (300/T)**b0 * EXP(c0/T)
-    ! because b0 = c0 = 0.  Therefore we can skip computing these
-    ! terms.  This avoids excess CPU cycles. (bmy, 12/18/20)
-    !
     REAL(dp) :: klo1,   klo2,   khi1,  khi2
     REAL(dp) :: xyrat1, xyrat2, blog1, blog2,   fexp1
     REAL(dp) :: fexp2,  kco1,   kco2,  TEMP300, k
     !
-    klo1   = 5.9E-33_dp * K300_OVER_TEMP
+    klo1   = 5.9E-33_dp * K300_OVER_TEMP**(1.0_dp)
```

Comment:
@msulprizio: Can we replace `K300_OVER_TEMP**(1.0_dp)` with just `K300_OVER_TEMP`?  This will be less likely to numerical noise, and will be less CPU cycles.

Reply:
Thanks. I had added that to reproduce the code in `tagged_co_mod.F90` exactly and ensure differences in CO were not originating in these lines, but I agree that it is not needed. It has been restored in https://github.com/geoschem/geos-chem/pull/2800/commits/f0d5265d97a7a91e275200c8c0b2f2d102ab085c.